.github/workflows/msrv.yml

```diff
@@ -10,61 +10,49 @@ jobs:
       matrix:
         rust:
           - 1.70.0
-          - stable
-          - beta
-          - nightly
 
     steps:
       - name: Checkout sources
         uses: actions/checkout@v4.1.1
 
+      - name: Cache
+        uses: Swatinem/rust-cache@v2
+
       - name: Install toolchain
         uses: dtolnay/rust-toolchain@master
         with:
           toolchain: ${{ matrix.rust }}
 
       - name: Run cargo check
-        if: matrix.rust != 'nightly'
-        run: cargo check --all-features
-
-      - name: Run cargo check (nightly)
-        if: matrix.rust == 'nightly'
-        continue-on-error: true
-        run: cargo check --all-features
+        run: cargo check --all-features --examples --tests
 
   test:
-    needs: [check]
     name: Test Suite
     runs-on: ubuntu-latest
     strategy:
       matrix:
         rust:
           - 1.70.0
-          - stable
+          - 1.73.0 # stable
```

Comment:
Probably should keep that as `stable`?

Reply:
No, having a mutable pointer to the current stable broke our CI before because new lints were introduced in stable rust that failed for our codebase. I'd rather have a non-mutable version here and update it manually every now and then.